xml/System.IO/FileSystemAclExtensions.xml

```diff
@@ -104,6 +135,37 @@ This extension method was added to .NET Core to bring the functionality that was
 
 This extension method was added to .NET Core to bring the functionality that was provided by the `System.IO.FileStream.#ctor(System.String,System.IO.FileMode,System.Security.AccessControl.FileSystemRights,System.IO.FileShare,System.Int32,System.IO.FileOptions,System.Security.AccessControl.FileSecurity)` .NET Framework constructor.
```

Comment:
same here 

BTW, the reason why I'm not using xref here is because we don't have a good way to link between monikers nowadays 
```suggestion
This extension method was added to .NET Core to bring the functionality that was provided by the [FileStream(String, FileMode, FileSystemRights, FileShare, Int32, FileOptions, FileSecurity)](https://docs.microsoft.com/dotnet/api/system.io.filestream.-ctor?view=netframework-4.8#System_IO_FileStream__ctor_System_String_System_IO_FileMode_System_Security_AccessControl_FileSystemRights_System_IO_FileShare_System_Int32_System_IO_FileOptions_System_Security_AccessControl_FileSecurity_) .NET Framework constructor.
```